app/scripts/lib/cropper.js

```diff
@@ -71,22 +74,28 @@ function (_) {
     this.slider.on('input', function () {
       self.resize(parseInt(this.value, 10));
     });
+    this.slider.on('change', function () {
+      self.metrics.logEvent('avatar-cropper.zoom.range');
```

Comment:
> Yes, good point. The view could pass in an event emitter (e.g. an object extended with Backbone event mixin) that listens for events and forwards them to our metrics. If the cropper receives such an object it'll trigger the events on that.

I can deal with that. Another options is to allow a consumer to pass in individual callbacks for rotate, zoom out and zoom in. The callbacks would log the events. Instead of requiring the mediator between DOM events and metrics be an EventEmitter, the mediator between the two would just be functions.